### PR TITLE
feat(chat): add attachment charts support (Issue #1021)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

Add attachment charts support

Issues:

- https://github.com/epam/ai-dial-chat/issues/1021

**UI changes:**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
         "node-fetch": "^3.3.2",
         "pino": "^8.15.1",
         "pino-pretty": "^10.2.0",
+        "plotly.js": "^2.30.0",
```
Comment (IlyaBondar) [inline package-lock.json:42]: make sure that you executed `npm ddp` (deduplication)

File: apps/chat/src/components/Chat/MessageAttachment.tsx [hunk 2]
```diff
@@ -79,6 +82,19 @@ const AttachmentDataRenderer = ({
       />
     );
   }
+  if (attachment.type === 'application/vnd.plotly.v1+json') {
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/MessageAttachment.tsx]: I suggest to make it openable
see on `isOpenable` and `imageTypes`

Comment (IlyaBondar) [reply apps/chat/src/components/Chat/MessageAttachment.tsx]: it should be in `AttachmentDataRenderer`

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1032.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1032.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by denys-kolomiitsev — APPROVED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1032.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**